File: backendSpring/src/main/kotlin/com/wizaord/moneyweb/domain/Account.kt
package com.wizaord.moneyweb.domain

import org.springframework.data.annotation.Id
import java.util.*

data class Account(
        @Id var id: String?,
        var name: String,
        var bankName: String,
        var openDate: Date,
        val isOpened: Boolean = false,
        var owners: MutableSet<AccountOwner> = mutableSetOf()
        )

File: frontend/src/app/domain/account/Account.ts
export class Account {
  accountName: string;
  bankName: string;
  dateCreate: Date;
  isOpened: boolean;
  solde: number;
  owners: string[];

  constructor(a: any) {
    this.accountName = a.accountName;
    this.bankName = a.bankName;
    this.dateCreate = new Date(a.dateCreate);
    this.isOpened = a.isOpened;
    this.owners = a.owners;
    this.solde = a.solde;
  }
}
